includes/definitions/discovery/icotera.yaml

```diff
@@ -0,0 +1,52 @@
+mib: ICOTERA-I6400-SERIES
+modules:
+    sensors:
+        voltage:
+            options:
+                index: 0
+                divisor: 10000
+            data:
+                -
+                    oid: transceiverDdmVoltage
+                    num_oid: '.1.3.6.1.4.1.29865.11.3.1.3.4.{{ $index }}'
+                    index: 'transceiverDdmVoltage.{{ $index }}'
+                    user_func: format_float
+                    descr: "WAN port transceiver"
+        current:
+            options:
+                index: 0
+                divisor: 1000
+            data:
+                -
+                    oid: transceiverDdmTxBiasCurrent
+                    num_oid: '.1.3.6.1.4.1.29865.11.3.1.3.5.{{ $index }}'
+                    index: 'transceiverDdmTxBiasCurrent.{{ $index }}'
+                    user_func: format_float
```

Comment:
Why do you want to limit the resolution of the data stored?

Reply:
To match the device's CLI output resolution.

Reply:
Please store the full precision and if you like, limit it in the webui.